cowprotocol/cowswap
# [2] feat(twap): add ComposableCowInfo to Order model

To distinguish `ComposableCoW` orders I've added `composableCowInfo` field to `Order` interface:

```
/**
 * https://github.com/rndlabs/composable-cow/blob/main/src/ComposableCoW.sol
 * Information about ComposableCoW conditional orders
 *
 * id - this parameter is specified when it's a conditional (parent) order
 * parentId - this parameter is specified when it's a discrete (child) order
 */
export type ComposableCowInfo = {
  id?: string
  parentId?: string
}
```

1. If an order has `order.composableCowInfo.id` it means this order is `ComposableCoW` order
2. If an order has `order.composableCowInfo.parentId` it means this order is a child of `ComposableCoW` order

See https://github.com/cowprotocol/cowswap/pull/2623 as an usage example

## To test
1. Orders table displaying
2. Orders cancellation (should properly change status: open -> cancelling -> cancelled)
